apps/chat/README.md

```diff
@@ -0,0 +1,240 @@
+
+## Assumptions
+
+- The developer should have their own AWS account. No preexisting set up is required.
+- The developer should have node.js installed to support running the Chime sample app
+  - Node.js can be downloaded here → https://nodejs.org/en/download/
+- ** IMPORTANT** : We currently only support us-east-1 so all the set-up must be done in us-east-1.
+
+## Cloudformation Deployment
+
+1. Ensure AWS CLI is installed
+    ```aws --version```
+2. Obtain AWS credentials for your AWS account
+3. Set the AWS credentials in the AWS CLI - https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.
+4. Deploy included Cloudformation template via [Cloudformation Console](https://aws.amazon.com/cloudformation/)
+    
+    or
+    
+    ```aws cloudformation create-stack --stack-name <STACKNAME> --template-body file://src/backend/serverless/template.yaml --parameters ParameterKey=DemoName,ParameterValue=<NAME_OF_DEMO> --capabilities CAPABILITY_NAMED_IAM CAPABILITY_AUTO_EXPAND```
+5. Verify and record outputs via [Cloudformation Console](https://aws.amazon.com/cloudformation/)
+    
+    or
+
+    ```aws cloudformation describe-stacks --stack-name <STACKNAME>```
+
+## Running the Amazon Chime Sample App
+
+1. Ensure your workspace has node.js installed. Type `node -v` in your terminal to confirm, and it should return a version number.
+2. Navigate to the root folder of Amazon Chime Sample App `apps/chat`
+3. In the root directory `apps/chat`, open `src/Config.js` with the editor of your choice and update the each missing config with the values from the
+ previously created resources.
+4. In the root directory `apps/chat`, run `npm install` and then `npm start` to build and start the client
+5. Open https://localhost:9000/ in your browser
+6. By default the demo uses Amazon Cognito User Pools to manage users and login.  Alternatively, you can get credentials through a small 
+ lambda service that the cloudformation template sets up.  If you prefer to use the default Cognito flow
+ continue with [Cognito User Pools - Register a New User](#cognito-user-pools-register-a-new-user).  If you prefer to use the credential 
+ exchange service see [Credential Exchange Service: Login](#Credential-Exchange-Service-Login).
+
+#### Cognito User Pools: Register a New User
+
+New users can register through the Amazon Chime Sample App.+ (nullable NSString *)serializeTimestamp:(NSDictionary *)rules value:(id)value error:(NSError *__autoreleasing *)error {
```

Comment:
This doesn't seem to be render correctly for me (The code shows in the right hand side)